### DISCUSSION TITLE
Aphrodite's bow goes where?

### DISCUSSION THREAD

**Discussion by Derphead** [ErinaSugino/Starbound-SBR-Mod#107] [Trouble Shooting] [answered]
This might be a dumb question, but I'm new to this. So far mods have been .paks that goes into my mod folder, but the Aphrodites bow is a folder with a bunch of folders in it. What do I do with it :D?

**Comment by ErinaSugino** [comment, marked as answer]
Treat the folder exactly as a .pak, cause that's what paks are.

For more precise instructions, [see here](https://github.com/ErinaSugino/Starbound-SBR-Mod/wiki/Advanced-Troubleshooting#step-1---installing-mods-correctly)
